serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -422,6 +422,88 @@ In your editor, change the first line in the `Dockerfile` entry where it reads `
 
 This image must be compatible with your operator's installation.
 
+[#deploy-supporting-services]
+== Deploy supporting services
```

Comment:
Can we move this section to its own file? Can be under Cloud > Operator > Supporting Services.

Then please do an introduction to the supporting services linking users for more info in Data Index > Core Concepts.

@nmirasch since the operator now supports deploying Data Index, we might need to review the Data index sections. Can you do it, please? 🙏

Reply:
done @ricardozanini